plyer/platforms/linux/screenshot.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+from plyer.facades import ScreenShot
+from plyer.utils import whereis_exe
+
+
+class GnomeScreenShot(ScreenShot):
```

Comment:
It is done to give to use the feature with which-ever utitlity is present. So need is to give separate names to both implemented classes. What say @KeyWeeUsr